app/settings/account/-components/security/component.ts

```diff
@@ -75,6 +104,26 @@ export default class SecurityPane extends Component {
         }
     });
 
+    @task({ withTestWaiter: true })
+    verifySecret = task(function *(this: SecurityPane) {
+        this.changeset.validate();
+        try {
+            if (this.changeset.isValid) {
+                yield this.changeset.save({});
+                this.showError = false;
```

Comment:
I think this should be done first thing in this task? At least it looks like that's how it used to work.

Reply:
I think the old `verifySecret` action was called when the save was successful, so I _think_ it still belongs in this block right here, after the changeset is saved. My understanding of ValidatedModelForm isn't the best though